PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,4 @@
+Fixes #issue number
```

Comment:
Create a separate Pull request for this change. 
Since this corresponds to other issue.
We want to keep each Pull request per issue to keep a track of them